excalidraw-app/debug.ts

```diff
@@ -131,5 +131,5 @@ export class Debug {
     };
   };
 }
-
+//@ts-ignore
```

Comment:
nice that someone in the DT repo for the `debug` package started polluting the global namespace or whatever is happening here :(